packages/storybook/stories/additional-docs.js

```diff
@@ -45,26 +70,79 @@ export const additionalDocs = {
     maturityCategory: USE,
     maturityLevel: BEST_PRACTICE,
   },
+  'Drop Down Panel': {
+    maturityCategory: CAUTION,
+    maturityLevel: AVAILABLE,
+  },
+  'Expanding Group': {
+    maturityCategory: CAUTION,
+    maturityLevel: AVAILABLE,
+  },
+  'Featured content': {
+    maturityCategory: USE,
+  },
+  'File Input': {
+    maturityCategory: CAUTION,
+    maturityLevel: AVAILABLE,
+  },
+  'Form': {
+    maturityCategory: USE,
+    maturityLevel: BEST_PRACTICE,
+  },
```

Comment:
I know it's like this on design.va.gov, but should we display maturity scale for parents? Form appears with a green icon in Storybook, but several of its nested components have use with caution or don't use.